frontends/bmd/src/app_root.tsx

```diff
@@ -1397,7 +1399,9 @@ export function AppRoot({
                 votes: votes ?? blankBallotVotes,
               }}
             >
-              <Ballot />
+              <RandomContext.Provider value={{ generator: randomGenerator() }}>
```

Comment:
This ensures we're using real random values from the browser's `crypto` API in real code.

Reply:
Why use a context instead of just a module/function mock?